hal/source/arg_pack.cpp

```diff
@@ -195,7 +201,11 @@ bool hal_argpack_t::expand(size_t num_bytes) {
         // copy from old to new and free old one
         std::memcpy(replacement_pack, pack, write_point);
       }
+#if defined(_MSC_VER)
+      free(pack);
```

Comment:
If I understand correctly, `_aligned_malloc` needs to be matched with `_aligned_free`.

Reply:
done